ui/src/views/compute/wizard/MultiNetworkSelection.vue

```diff
@@ -280,6 +280,16 @@ export default {
       }
       this.sendValuesTimed()
     },
+    getDefaultNetwork (record) {
+      if (record.vlanid && record.vlanid !== -1) {
+        const matched = this.validNetworks[record.id].filter(x => Number(x.vlan) === record.vlanid)
```

Comment:
@abh1sar 
x.vlan looks like `vlan://2472`, but record.vlanid looks like `2472`, or vice versa

can you have a look ?